### PR TITLE
[tools-atip] Add new steps for upgrade app on android

### PR DESCRIPTION
**Pull request by wanghongjuan**
- The step can run normally

Impacted TCs num(approved): New 0, Update 1, Delete 0
Unit test Platform: Android crosswalk-16.45.419.0
Android test result summary: Pass 1, Fail 0, Block 0
BUG=https://crosswalk-project.org/jira/browse/XWALK-4844

### CODE REVIEW

File: tools/atip/atip/common/steps.py [hunk 1]
```diff
@@ -195,6 +195,16 @@ def install_app_by_cmd(context, app_path):
             device_id = context.bdd_config["platform"]["device"]
         assert context.web.install_app_by_cmd(device_id, app_path)
 
+# reinstall android app by commend line
+@step(u'reinstall "{app_path}"')
```
Comment (Honry) [inline tools/atip/atip/common/steps.py]: 's/reinstall/upgrade'

Comment (Honry) [inline tools/atip/atip/common/steps.py]: Please also update the README.md file.

**Comment by wanghongjuan** [comment]
Updated

**Comment by Honry** [comment]
Please also update the README.md file.

**Comment by Honry** [comment]
LGTM

**Pull request merged**